dotCMS/src/integration-test/java/com/dotmarketing/util/ConfigTest.java

```diff
@@ -0,0 +1,254 @@
+    private static final Map<String, String> envMap = new HashMap<>();
+
+    final static String DOT_TESTING_INTEGER="DOT_TESTING_INTEGER";
+    final static String XXX_TESTING_INTEGER = "XXX_TESTING_INTEGER";
+    final static String DOT_TESTING_LONG= "DOT_TESTING_LONG";
+    final static String DOT_TESTING_BOOLEAN ="DOT_TESTING_BOOLEAN";
+    final static String DOT_TESTING_FLOAT ="DOT_TESTING_FLOAT";
+    
+    final static String SIMPLE_TESTING_STRING = "SIMPLE_TESTING_STRING";
+    final static String TESTING_VALUE = "TESTING_VALUE";
+    final static String[] TESTING_ARRAY = new String[]{"TESTING_VALUE"};
+    final static String NO_VALUE = "NO_VALUE";
+    final static String DOT_TESTING_STRING_WITH_COMMA ="DOT_TESTING_STRING_WITH_COMMA";
+    final static String DOT_TESTING_STRING_WITH_SPACES ="DOT_TESTING_STRING_WITH_SPACES";
+    final static String DOT_TESTING_STRING ="DOT_TESTING_STRING";
+    final static String UNABLE_TO_READ_VAR="UNABLE_TO_READ_VAR";
+    
+    
+    
+    /**
+     * This method sets environmental variables that are intended for testing.
+     * 
+     */
+    private static void setTestEnvVariables() {
+        
+        System.getenv().put(DOT_TESTING_INTEGER, String.valueOf(Integer.MAX_VALUE));
+        System.getenv().put(XXX_TESTING_INTEGER, String.valueOf(Integer.MIN_VALUE));
+        System.getenv().put(DOT_TESTING_LONG, String.valueOf(Long.MAX_VALUE));
+        System.getenv().put(DOT_TESTING_BOOLEAN, String.valueOf(Boolean.TRUE));
+        System.getenv().put(DOT_TESTING_FLOAT, String.valueOf(Float.MAX_VALUE));
+        System.getenv().put(DOT_TESTING_STRING_WITH_COMMA, "VALUE1,VALUE2");
+        System.getenv().put(DOT_TESTING_STRING_WITH_SPACES, "VALUE1 VALUE2");
+        System.getenv().put(DOT_TESTING_STRING, "VALUE_ABC");
+        System.getenv().put(UNABLE_TO_READ_VAR, "NOPE");
+    }
+    
+    
+    
+    @Test
+    public void testing_string_with_comma() {
+
+        String value = Config.getStringProperty("TESTING_STRING_WITH_COMMA");
+        assert(value.equals(Config.getStringProperty("testing.String.with_comma")));
+
+        
+    }
+    
+    
+    
+    @Test
+    public void testing_null_returns() {
+
+        String test = Config.getStringProperty(SIMPLE_TESTING_STRING, null);
+        assertNull(test);
+
+        
+    }
+    
+    
+    @Test
+    public void testing_default_returns() {
+
+        
+        String test = Config.getStringProperty(SIMPLE_TESTING_STRING, TESTING_VALUE);
+        assertEquals(test, TESTING_VALUE);
+
+        
+    }
+    
+    @Test
+    public void testing_notfound_string_returns() {
+
+
+        assert(Config.getStringProperty("no-property") ==null);
+  
+
+
+        
+    }
+
+
+    
+    @Test
+    public void testing_notfound_int_returns() {
+
+        try {
+            Config.getIntProperty("no-property");
+            assert(false);
+        }
+        catch(Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5943250)